### PR TITLE
Fdg 5920 - G-Develop Initial AFG Overview Page

### PR DESCRIPTION
**Pull request by jsmend0za-frb**
The new AFG Overview page is created using the Zeplin design mock, adding key page elements such as SEO title/description/H1, Table of Contents, sections headings, dummy text, images, and DS&M. Note for the Year-in-Review section - the header copy was changed since the mock was created. The mock can still be followed for font size/coloring. For the use of emphasis coloring, for Revenue that color can be applied to the word ‘collect’;  for Spending the word ‘spent’; for Deficit the word ‘deficit’; and for Debt the word ‘debt’.

A placeholder is added for the social share component since that work will be completed later with [FDG-5921](https://federal-spending-transparency.atlassian.net/browse/FDG-5921).

The URL for the new page is: https://fiscaldata.treasury.gov/americas-finance-guide/

Ensure on mobile that the copy on the Statue of Liberty image is clear (timebox this).

The page is responsive for desktop and mobile devices.

Code Coverage: 89.41

https://federal-spending-transparency.atlassian.net/jira/software/c/projects/FDG/boards/44?modal=detail&selectedIssue=FDG-5920&assignee=6304f22ca29ccf49383889f2&assignee=62fbed9be2b1b5ae159aa4c9

### CODE REVIEW

File: src/pages/americas-finance-guide/index.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import CompareSection from './afg-components/compare-section/compare-section';
+
+export default function AmericasFinanceGuidePage() {
+
+
+
+  return (
+    <SiteLayout isPreProd={false}>
+      <PageHelmet
+        pageTitle="America’s Finance Guide | U.S. Treasury Fiscal Data"
```
Comment (j-hertzler) [inline src/pages/americas-finance-guide/index.jsx:20]: U.S. Treasury Fiscal Data can be removed here

File: src/pages/americas-finance-guide/afg-components/compare-section/compare-section.module.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+@import '../../../../variables.module.scss';
+
+.compareSection{
+    
+    margin-top: 4rem;
+    margin-bottom: 4rem;
+
+    // .compareGridContainer{
```
Comment (j-hertzler) [inline src/pages/americas-finance-guide/afg-components/compare-section/compare-section.module.scss:8]: Remove commented out code

File: src/pages/americas-finance-guide/afg-overview.module.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
+@import '../../variables.module.scss';
+
+.topContainer {
+    max-width: $breakpoint-lg;
+}
+
+.debtText {
+    color: #b3532d
```
Comment (j-hertzler) [inline src/pages/americas-finance-guide/afg-overview.module.scss:8]: Update to use color variables where possible

**Review by j-hertzler — CHANGES_REQUESTED** [review]
The changes look good and address most comments. 
New / remaining issues : 
- Learn more about federal spending link is the wrong color
- On mobile, images should be above their section header, rather than below (example of this is in the image previously added to this pr)

**Comment by jsmend0za-frb** [comment]
I have address these comments and should be fixed now. Thanks.

**Review by j-hertzler — APPROVED** [review]
Changes look good. Tests and build pass locally

**Pull request merged**